subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -108,6 +108,21 @@ Previously Gradle used to detect if an input file collection annotated with `@Sk
 To ignore directories in Gradle 8.0 and later, the input property needs to be explicitly annotated with `@IgnoreEmptyDirectories`.
 For more information see <<empty_directories_file_tree>>.
 
+==== Project facets generated by the `eclipse wtp' plugin have changed
+
+A `jst.utility` facet was added to java projects by the `eclipse wtp` plugin. 
+This facet will now be removed when you add a `jst.ejb` facet: 
```

Comment:
```suggestion
The `eclipse wtp` plugin adds the `jst.utility` facet to java projects.
Now, adding the `jst.ejb` facet implicitly removes the `jst.utility` facet: 
```
Nitpick: use active voice and simpler phrasing when possible.
Suggestion: it might be useful for readers to know why we avoid combining these facets. Consider adding a glue sentence in between that explains that motivation -- "But using the `jst.ejb` and `jst.utility` facets in the same projects led to _____." -- or similar.